frontend/src/metabase/visualizations/components/settings/ChartSettingColorsPicker.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component, PropTypes } from "react";
+
+import { normal } from 'metabase/lib/colors'
+const DEFAULT_COLOR_HARMONY = Object.values(normal);
```

Comment:
I've noticed this pattern in a couple of places. Wondering if we should maybe make the default export the array of colors so we don't have to always do this to get the hex values.